src/preamble.js

```diff
@@ -1764,6 +1764,7 @@ var Math_imul = Math.imul;
 var Math_fround = Math.fround;
 var Math_min = Math.min;
 var Math_clz32 = Math.clz32;
```

Comment:
since `trunc` is fairly new in ES6, we should have a [pollyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc).